newIDE/app/src/ProjectManager/index.js

```diff
@@ -789,472 +841,622 @@ export default class ProjectManager extends React.Component<Props, State> {
-                      onAdd={() => this._addEventsFunctionsExtension(i)}
-                      onRename={newName => {
-                        this.props.onRenameEventsFunctionsExtension(
-                          name,
-                          newName
-                        );
-                        this._onEditName(null, '');
-                      }}
-                      onEditName={() =>
-                        this._onEditName('events-functions-extension', name)
-                      }
-                      onCopy={() =>
-                        this._copyEventsFunctionsExtension(
-                          eventsFunctionsExtension
-                        )
-                      }
-                      onCut={() =>
-                        this._cutEventsFunctionsExtension(
-                          eventsFunctionsExtension
-                        )
-                      }
-                      onPaste={() => this._pasteEventsFunctionsExtension(i)}
-                      onDuplicate={() =>
-                        this._duplicateEventsFunctionsExtension(
-                          eventsFunctionsExtension,
-                          i
-                        )
-                      }
-                      canPaste={() =>
-                        Clipboard.has(EVENTS_FUNCTIONS_EXTENSION_CLIPBOARD_KIND)
-                      }
-                      canMoveUp={i !== 0}
-                      onMoveUp={() => this._moveUpEventsFunctionsExtension(i)}
-                      canMoveDown={
-                        i !== project.getEventsFunctionsExtensionsCount() - 1
-                      }
-                      onMoveDown={() =>
-                        this._moveDownEventsFunctionsExtension(i)
-                      }
+                        }
+                        initiallyOpen={false}
+                        autoGenerateNestedIndicator={true}
+                        indentNestedItems
+                        renderNestedItems={() => [
+                          <ListItem
+                            key="properties"
+                            primaryText={<Trans>Properties</Trans>}
+                            leftIcon={<SettingsApplications />}
+                            onClick={this._openProjectProperties}
+                          />,
+                          <ListItem
+                            key="global-variables"
+                            primaryText={<Trans>Global variables</Trans>}
+                            leftIcon={<VariableTree />}
+                            onClick={this._openProjectVariables}
+                          />,
+                          <ListItem
+                            key="icons"
+                            primaryText={<Trans>Icons</Trans>}
+                            leftIcon={<PhotoLibrary />}
+                            onClick={() =>
+                              this.props.onOpenPlatformSpecificAssets()
```

Comment:
I think all those onClick can be simplified to not create a function every time: `onClick={this.props.onOpenPlatformSpecificAssets}`

Reply:
True, I'll change it